packages/core/src/components/input/number-input.tsx

```diff
@@ -178,8 +180,15 @@ export class NumberInput implements IxInputFieldComponent<number> {
     this.updateFormInternalValue(this.value);
   }
 
-  componentDidRender() {
-    this.updatePaddings();
+  componentDidLoad() {
```

Comment:
```suggestion
  componentDidRender() {
```

This was changed from 'componentDidLoad()' to 'componentDidRender()' in both 'input' and 'date-input'.

Reply:
Nice catch!